xml/index.xml

```diff
@@ -54,6 +54,15 @@ with fixes for
 </para>
 </event>
 
+<event date="2024-05-13">
```

Comment:
Posting outdated news looks nonsensical, no one will read it. 
Also it has low value for nginx.org.
Better make an actual news entry and mention all the repos altogether.

Reply:
The news part moved to a separate commit, pls see the new version.